### ISSUE TITLE
Typo @Mat-Select API Docs?

### ISSUE THREAD

**Issue by Maxstgt** [angular/material.angular.io#676]
Bug, enhancement request, or proposal:
I guess, there is a typo on the angular material docs.
![Screenshot from 2019-11-12 11-34-37](https://user-images.githubusercontent.com/39312833/68685406-622cd900-056a-11ea-8fb2-d219c90360e4.png)

Shouldn't it be panel?

Detailed Reproduction Steps:
Visit https://material.angular.io/components/select/api
What is the use-case or motivation for changing an existing behavior?
To make the world typo-free ;)

**Comment by Splaktar** [comment]
Yeah, I guess it should technically be `panel` instead of `pane`. I think that this comment/doc should also explain the naming of `overlayDir`, what does `Dir` mean here?

I'm moving this to the correct repo for Angular Material issues. This repo is for docs infrastructure only.
